src/components/PrizePage/Prizes.js

```diff
@@ -64,11 +71,44 @@ const TableCell = withStyles({
 })(MuiTableCell);
 
 const useStyles = makeStyles({
+	prizeGrid: {
+		textAlign: 'center'
+	},
+	prizeCard: {
+		backgroundColor: '#e6e6e6',
+		borderRadius: '10px',
+		height: '100%',
+		paddingTop: `${prizeImageRadius}`,
+		paddingBottom: '20px',
+		position: 'relative',
+		zIndex: '-1'
+	},
+	prizeDescription: {
+		paddingBottom: '20px'
+	},
 	header: {
 		fontWeight: 700,
 		textDecoration: 'underline',
 		fontSize: '1.5em'
 	},
+	imageBox: {
+		alignItems: 'center',
+		alignSelf: 'center',
+		backgroundColor: '#fafafa',
+		border: '6px solid',
+		borderColor: '#e6e6e6',
+		borderRadius: prizeImageRadius,
+		display: 'flex',
+		justifyContent: 'center',
+		marginBottom: `calc(-1 * ${prizeImageRadius})`,
```

Comment:
Either go full CSS and use `'--radius': '50px'`, or go full JavaScript. You can let `prizeImageRadius` be `50`, and then set marginBottom to `-prizeImageRadius` in JavaScript.